serveradmin/servershell/views.py

```diff
@@ -274,12 +276,15 @@ def _edit(request, server, edit_mode=False, template='edit'):  # NOQA: C901
                 except ValidationError:
                     invalid_attrs.add(attribute_id)
 
-            # TODO: Remove when PR135 is merged
-            if type(current[attribute_id]) in (IPv4Address, IPv6Address):
-                current[attribute_id] = ip_interface(current[attribute_id])
+            if current:
```

Comment:
You might want to check if you want
``` python
if attribute_id in current:
    ...
```
Just to prevent a KeyError two lines later. Possible that you want that KeyError though.

Reply:
The key error could still happen for new objects when we rely on the client but I think it is still an improvement. Will add.